vagrant/provisioning/roles/arkcase-app/templates/arkcase-commons-server.yaml

```diff
@@ -1,3 +1,8 @@
 arkcase:
   kafka:
     bootstrapAddress: {{ kafka_host }}:{{ kafka_port | default(9092) }}
+    schemaRegistryAddress: https://{{ internal_host }}/schema-registry
```

Comment:
Will the schema registry always be on the same host as this commons-server?  Or would it be on the `kafka_host` or some other host?

Reply:
I will introduce variables for each microservice like arkcase_schema_host, arkcase_api_gateway etc., and use internal_host as default